e2e/tests/test_e2e.py

```diff
@@ -863,10 +913,7 @@ def get_pg_nodes(self, pg_cluster_name, namespace='default'):
         return master_pod_node, replica_pod_nodes
 
     def wait_for_operator_pod_start(self):
-        self. wait_for_pod_start("name=postgres-operator")
-        # HACK operator must register CRD and/or Sync existing PG clusters after start up
-        # for local execution ~ 10 seconds suffices
-        time.sleep(60)
+        self. wait_for_pod_start("name=postgres-operator")        
```

Comment:
And how do we know then at  [the line 109](https://github.com/zalando/postgres-operator/blob/4fc8ca384d6485fcb0756f7cfeccd40a956101af/e2e/tests/test_e2e.py#L109) that it is safe to create PG clusters ? By re-creating a PG cluster until it succeeds ?

Reply:
you worry the operator is not running? and then initial manifest create is not picked up?

Reply:
it might happen that the PG CRD is not yet registered. There is a very short period of time when an operator pod has already started by the CRD is not yet registered. That sometimed led to the failure of cluster creation during the initial development of e2e tests